### PR TITLE
Update index.php of Cassiopeia, so that it is easier to use a child of Cassiopeia

### PR DESCRIPTION
**Pull request by astridx**
If I'm not mistaking it's more straightforward to create a child template with this change.

### CODE REVIEW

File: templates/cassiopeia/index.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
 // Color Theme
 $paramsColorName = $this->params->get('colorName', 'colors_standard');
 $assetColorName  = 'theme.' . $paramsColorName;
-$wa->registerAndUseStyle($assetColorName, 'media/templates/site/cassiopeia/css/global/' . $paramsColorName . '.css');
+$wa->registerAndUseStyle($assetColorName, 'media/templates/site/' . $this->template . '/css/global/' . $paramsColorName . '.css');
```
Comment (dgrammatiko) [inline templates/cassiopeia/index.php:42]: This breaks the cascade, ie if the file `x.css` doesn't exist in the child and it exists in the parent then load it from there. I haven't tested but I think the code here should be 
```php
$wa->registerAndUseStyle($assetColorName, 'global/' . $paramsColorName . '.css');

```

(maybe it needs an attribute `['relative' => true]` to work correctly, but it should be relative path so the cascade is not broken)

**Comment by richard67** [comment]
We should keep things consistent which are the same in index.php, error.php and maybe also component.php and offline.php. I am almost sure that the code changed here does not only exist in index.php but also in some of the other mentioned files.

**Comment by astridx** [comment]
> This breaks the cascade, ie if the file `x.css` doesn't exist in the child and it exists in the parent then load it from there. I haven't tested but I think the code here should be
> 
> ```
> $wa->registerAndUseStyle($assetColorName, 'global/' . $paramsColorName . '.css');
> ```
> 
> (maybe it needs an attribute `['relative' => true]` to work correctly, but it should be relative path so the cascade is not broken)

Oops, you're right. I hadn't considered that not all child templates overwrite all SCSS/CSS data.

**Comment by dgrammatiko** [comment]
@astridx this is not an invalid issue, the solution needs just some changes

**Pull request closed without merging**